A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/CDU/OPTIONS/A320_Neo_CDU_OPTIONS_AOC_SIMBRIEF.js

```diff
@@ -20,22 +20,18 @@ class CDU_OPTIONS_SIMBRIEF {
     static ShowPage(mcdu) {
         mcdu.clearDisplay();
 
-        const simbriefUsername = NXDataStore.get("CONFIG_SIMBRIEF_USERNAME", "").replace(/_/g, ' ');
+        const simbriefUser = NXDataStore.get("CONFIG_SIMBRIEF_USERID", "");
 
-        const simbriefUsernameString = simbriefUsername ? `{green}[${simbriefUsername}]{end}` : "{cyan}*[\xa0\xa0\xa0\xa0\xa0]{end}";
-
-        const simbriefUserId = NXDataStore.get("CONFIG_SIMBRIEF_USERID", "");
-
-        const simbriefUserIdString = simbriefUserId ? `{green}[${simbriefUserId}]{end}` : "{cyan}*[\xa0\xa0\xa0\xa0\xa0]{end}";
+        const simbriefUserString = simbriefUser ? `{cyan}\xa0${simbriefUser}{end}` : "{amber}[\xa0\xa0\xa0\xa0\xa0\xa0]{end}";
```

Comment:
We wouldn't usually use amber for optional fields, cyan is the correct color. Dropping the star is fine though.

Reply:
Fixed.